src/availability/query_data.rs

```diff
@@ -213,23 +213,25 @@ impl<Types: NodeType> LeafQueryData<Types> {
         leaf: Leaf<Types>,
         qc: QuorumCertificate<Types>,
     ) -> Result<Self, InconsistentLeafError<Types>> {
+        // TODO: Replace with the new `commit`.
+        let leaf_commit = <Leaf<Types> as Committable>::commit(&leaf);
```

Comment:
What is the new function that we need to call for the new commit? I think some more comments for that would be great

Reply:
It's just `leaf.commit`. I can add a more detailed comment.